src/Features/LanguageServer/ProtocolUnitTests/HandlerTests.cs

```diff
@@ -179,17 +180,15 @@ public Task<string> HandleRequestAsync(RequestContext context, CancellationToken
             }
         }
 
-        [ExportCSharpVisualBasicStatelessLspService(typeof(NotificationHandler)), PartNotDiscoverable, Shared]
         [LanguageServerEndpoint(MethodName, LanguageServerConstants.DefaultLanguageName)]
         internal class NotificationHandler : ILspServiceNotificationHandler<TestRequestTypeOne>
```

Comment:
```suggestion
        internal sealed class NotificationHandler : ILspServiceNotificationHandler<TestRequestTypeOne>
```

Reply:
this was confusing.  can you rename to TestNotificationHandler to make it clear at the use site that the test is validating this particular handler impl